code/modules/mob/living/carbon/carbon.dm

```diff
@@ -797,7 +797,7 @@
 		B.brain_death = FALSE
 	for(var/thing in diseases)
 		var/datum/disease/D = thing
-		if(D.severity != DISEASE_SEVERITY_POSITIVE)
+		if(!D.severity == DISEASE_SEVERITY_BENEFICIAL && !D.severity == DISEASE_SEVERITY_POSITIVE)

```

Comment:
Why we doing this instead of !=

Reply:
that's my bad, never had to use != before so it slipped my mind